src/diffpy/pdfmorph/tests/test_morphshape.py

```diff
@@ -6,14 +6,16 @@
 import numpy
 import pytest
 
+from diffpy.pdfmorph.morphs.morphishape import MorphISpheroid
+from diffpy.pdfmorph.morphs.morphshape import MorphSphere, MorphSpheroid
+
+# FIXME: add MorphISphere test
+
```

Comment:
Add FIXME after removing import. Theoretically we should require a test for `MorphISphere` as well.